Mathmagician/Fibonacci.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class Fibonacci
+    {
+        //contains the first two fibonaccis, ready to go
+        public List<int> FirstTwoInSequence = new List<int> {1, 1};
+
+        //list to store 
+        public List<int> Fibonaccis = new List<int>();
+
+        //this method takes an int argument.
+        //if the number of requested fibonaccis is less than 3,
+        //it simply prints off the first or first two numbers in the 'sequence' list.
+        //ask for more than 3, and it will add 1 and 1 to the list, 
+        //start generatin' fibbos, add the generated numbers to the list, rinse & repeat 
+        public void GenerateFibonacci(int numOfFibonacci)
+        {
+            if (numOfFibonacci == 0 )
```

Comment:
You should return a list out of this method and have something else print to the console.  This method is doing too much.